components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -50,8 +50,17 @@ app-page-picker {
     margin-bottom: .6em;
   }
 
+  .channel-line {
```

Comment:
This new style is exactly as `service-line`, maybe renaming it to `data-line` and using it multiple times would be my advice.